core/controllers/feedback.py

```diff
@@ -24,11 +24,13 @@
 
 transaction_services = models.Registry.import_transaction_services()
 
+
 class ThreadListHandler(base.BaseHandler):
     """Handles operations relating to feedback thread lists."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    @acl_decorators.open_access
```

Comment:
When a user gives feedback while viewing exploration, it goes through `ReaderFeedbackHandler` in reader.py